app/experimenter/nimbus-ui/src/components/PageHome/index.stories.tsx

```diff
@@ -41,6 +41,20 @@ export const Basic = storyTemplate([mockDirectoryExperimentsQuery()]);
 
 export const Loading = storyTemplate([]);
 
+export const QueryError = () => {
+  const mockWithError = {
+    ...mockDirectoryExperimentsQuery(),
+    error: new Error("boop, something's actually wrong"),
+  };
+  return (
+    <MockedCache mocks={[mockWithError, mockWithError]}>
```

Comment:
FWIW I think we have room for a little cleanup here and I noticed the "Loading" story hasn't worked since we started handling the errors; filed #6113 as a follow up maintenance task.